xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-ui/src/main/java/org/xwiki/test/ui/po/AbstractRegistrationPage.java

```diff
@@ -65,7 +65,7 @@ public void fillInJohnSmithValues()
     public void fillRegisterForm(final String firstName, final String lastName, final String username,
         final String password, final String confirmPassword, final String email)
     {
-        Map<String, String> map = new LinkedHashMap<String, String>();
+        LinkedHashMap<String, String> map = new LinkedHashMap<String, String>();
```

Comment:
Why did you changed that?

Reply:
Reverted :+1: 
 I thought I could use some API from this class but in the end it wasn't useful.